src/x/instrument/build.go

```diff
@@ -52,6 +53,10 @@ var (
 	// set to a non-empty string, we log the build information at process startup.
 	LogBuildInfoAtStartup string
 
+	// LogBuildInfoToStdout controls whether we log build information to stdout or stderr.
+	// If it and LogBuildInfoAtStartup are set to a non-empty string
+	LogBuildInfoToStdout string
```

Comment:
lol `LogBuildInfoToStdout=false` -> true

Reply:
Yea, this is admittedly kinda weird, but I wanted to keep it consistent with `LogBuildInfoAtStartup`